.github/no-response.yml

```diff
@@ -0,0 +1,2 @@
+daysUntilClose: 30
+responseRequiredLabel: needs-input
```

Comment:
Shouldn't we also provide a short comment via `closeComment`?

Reply:
The default comment is pretty good imo:

```
  This issue has been automatically closed because there has been no response
  to our request for more information from the original author. With only the
  information that is currently in the issue, we don't have enough information
  to take action. Please reach out if you have or find the answers we need so
  that we can investigate further.
```
(from https://probot.github.io/apps/no-response/)

Reply:
✅